microhapdb/tests/test_frequency.py

```diff
@@ -86,3 +86,12 @@ def test_all_sources(marker, pop, allele, frequency):
     result = freq[(freq.Marker == marker) & (freq.Population == pop) & (freq.Allele == allele)]
     assert len(result) == 1
     assert result.Frequency.iloc[0] == pytest.approx(frequency)
+
+
+def test_all_consistent():
+    inconsistent_markers = set()
+    for markerid, table in microhapdb.frequencies.groupby("Marker"):
+        numvars = set([a.count("|") + 1 for a in table.Allele])
+        if len(numvars) > 1:
+            inconsistent_markers.add(markerid)
+    assert len(inconsistent_markers) == 0, sorted(inconsistent_markers)
```

Comment:
After fixing the aforementioned problem, I wrote this test and ran it iteratively to identify and fix additional issues.